_news/2018-06-07-published-gcp-seurity-best-practices.md

```diff
@@ -0,0 +1,9 @@
+---
+title: GCP Security Best Practices
+author: Justin Jawad
+---
+Forseti Security provides security assurance for your GCP environment. Check out the links below to learn more about best practices around securing GCP environments from the team @ Google. 
+
+* [IAM Security Best Practices](https://cloud.google.com/iam/docs/using-iam-securely)
+
```

Comment:
Ah, I was hoping we could introduce our own best practices (/concepts/best-practices.html) here. Can you rewrite that to do this? Maybe in that case we can make @MartenvanWezel the author as he's updating it.